pkg/kubectl/plugins/plugins.go

```diff
@@ -91,12 +96,29 @@ type Plugins []*Plugin
 
 // Flag describes a single flag supported by a given plugin.
 type Flag struct {
-	Name      string `json:"name"`
-	Shorthand string `json:"shorthand,omitempty"`
-	Desc      string `json:"desc"`
-	DefValue  string `json:"defValue,omitempty"`
+	Name      string   `json:"name"`
+	Type      FlagType `json:"type"`
```

Comment:
To make it backwards compatible, this new-added field should be optional.

Should be `json:"type, omitempty"` here.